README.md

```diff
@@ -0,0 +1,84 @@
+# LBRY Block Explorer
+
+A simple PHP block explorer for browsing transactions and claims on the [LBRY](https://lbry.io) blockchain. The explorer was developed using CakePHP which is a model-view-controller (MVC) PHP framework.
```

Comment:
"which is a model-view-controller (MVC) PHP framework" could probably be removed, but not a strong opinion